--- js/src/jit/none/Lowering-none.h
+++ js/src/jit/none/Lowering-none.h
@@ -64,6 +64,7 @@
     void lowerMulI(MMul *, MDefinition *, MDefinition *) { MOZ_CRASH(); }
     void lowerUDiv(MDiv *) { MOZ_CRASH(); }
     void lowerUMod(MMod *) { MOZ_CRASH(); }
+    void visitValue(LValue *value) { MOZ_CRASH(); }
     void visitBox(MBox *box) { MOZ_CRASH(); }
     void visitUnbox(MUnbox *unbox) { MOZ_CRASH(); }
     void visitReturn(MReturn *ret) { MOZ_CRASH(); }
@@ -77,6 +78,7 @@
     void visitAsmJSStoreHeap(MAsmJSStoreHeap *ins) { MOZ_CRASH(); }
     void visitAsmJSLoadFuncPtr(MAsmJSLoadFuncPtr *ins) { MOZ_CRASH(); }
     void visitStoreTypedArrayElementStatic(MStoreTypedArrayElementStatic *ins) { MOZ_CRASH(); }
+    void visitSubstr(MSubstr *ins) { MOZ_CRASH(); }
     void visitAtomicTypedArrayElementBinop(MAtomicTypedArrayElementBinop *ins) { MOZ_CRASH(); }
     void visitCompareExchangeTypedArrayElement(MCompareExchangeTypedArrayElement *ins) { MOZ_CRASH(); }
     void visitAsmJSCompareExchangeHeap(MAsmJSCompareExchangeHeap *ins) { MOZ_CRASH(); }
