From 8b6e2070844f78868e57ab95195003bc0c5a7020 Mon Sep 17 00:00:00 2001
From: "Jory A. Pratt" <anarchy@gentoo.org>
Date: Fri, 15 Jan 2016 21:56:30 -0600
Subject: [PATCH 1/2] Prevent the sdk from being installed from here on out

Signed-off-by: Jory A. Pratt <anarchy@gentoo.org>
---
 browser/installer/Makefile.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/browser/installer/Makefile.in b/browser/installer/Makefile.in
index 8722799..3c4624b 100644
--- a/browser/installer/Makefile.in
+++ b/browser/installer/Makefile.in
@@ -95,8 +95,6 @@ MOZ_PKG_MAC_ICON=branding/disk.icns
 MOZ_PKG_MAC_EXTRA=--symlink '/Applications:/ '
 endif
 
-INSTALL_SDK = 1
-
 include $(topsrcdir)/toolkit/mozapps/installer/signing.mk
 include $(topsrcdir)/toolkit/mozapps/installer/packager.mk
 
-- 
2.7.0

