--- a/js/src/jit/AtomicOperations.h	2016-06-06 16:15:02.000000000 -0400
+++ b/js/src/jit/AtomicOperations.h	2016-06-27 13:40:39.261569170 -0400
@@ -339,6 +339,8 @@
 #  include "jit/none/AtomicOperations-ppc.h"
 # elif defined(__aarch64__)
 #  include "jit/arm64/AtomicOperations-arm64.h"
+# elif defined(__i386__) || defined(__x86_64__)
+#  include "jit/x86-shared/AtomicOperations-x86-shared.h"
 # else
 #  include "jit/none/AtomicOperations-none.h" // These MOZ_CRASH() always
 # endif
